rllib/models/trajectory_views.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
```

Comment:
Helper function to generate input_dicts (Views) from view requirements each time a forward pass is done through the model.